go.mod

```diff
@@ -14,19 +14,19 @@ require (
 	github.com/cloudflare/cloudflare-go v0.14.0
 	github.com/cockroachdb/pebble v0.0.0-20230209160836-829675f94811
 	github.com/consensys/gnark-crypto v0.12.1
-	github.com/crate-crypto/go-ipa v0.0.0-20231025140028-3c0104f4b233
+	github.com/crate-crypto/go-ipa v0.0.0-20240223125850-b1e8a79f509c
```

Comment:
For now, I'm sitting on top of https://github.com/ethereum/go-verkle/pull/429, but when that is merged I'll change it to master.

Reply:
Done